src/templates/EventList.html

```diff
@@ -1,4 +1,4 @@
-<div id="event-list" :class="['event-list', { '-active': view === 'list'}]">
+<div id="event-list" class="event-list" :class="['event-list', { '-active': view === 'list'}]">
```

Comment:
So here are two things: the initial classes to be parsed by the browser, and the vue-bound classes to be used when vue takes over the DOM later in the cycle.